docs/advanced/additional-requests.rst

```diff
@@ -304,7 +308,7 @@ Executing a HttpRequest instance
             return item
 
 As the example suggests, we're performing an additional request that allows us
-to extract more images in a product page that might not be otherwise be possible.
+to extract more images in a product page that MAY NOT be otherwise be possible.
```

Comment:
I think in cases like this we shouldn't follow RFC terms. Here we're just explaining the example. I think RFC applies where we explain the rules user needs to follow to implement something, and where we need to be really specific on what's optional and what's not optional.

Reply:
Reverted some of the updated terms in https://github.com/scrapinghub/web-poet/pull/38/commits/29aa8cc6d81fec598451e8493e91045661e9d9b8.